src/moin/scripts/migration/moin19/import19.py

```diff
@@ -65,10 +63,12 @@
     'creole': 'text/x.moin.creole;charset=utf-8',
     'python': 'text/x-python;charset=utf-8',
     'text/creole': 'text/x.moin.creole;charset=utf-8',
-    'rst': 'text/rst;charset=utf-8',
-    'text/rst': 'text/rst;charset=utf-8',
+    'rst': 'text/x-rst;charset=utf-8',
+    'text/rst': 'text/x-rst;charset=utf-8',
```

Comment:
hmm, correct.

strange, wondering how we found the wrong mimetype...

Reply:
Probably never tested, or not tested recently